superchain/configs/sepolia-seal-0/sealchain-1.yaml

```diff
@@ -0,0 +1,22 @@
+name: sealchain-1
+chain_id: 18714
+public_rpc: https://rpc-seal-rollup-op-6hhwuguskm.t.conduit.xyz
+sequencer_rpc: http://awe.some.seq.rpc
```

Comment:
This is from the `.env.example`: `SEQUENCER_RPC="http://awe.some.seq.rpc"`. I don't know if that was intended, but I imagine this should be filled out.